hat/assets/js/apps/Iaso/domains/storages/hooks/useGetOperationsTypes.ts

```diff
@@ -0,0 +1,55 @@
+// @ts-ignore
+import { useSafeIntl } from 'bluesquare-components';
+import { IntlFormatMessage } from '../../../types/intl';
+
+import { DropdownOptions } from '../../../types/utils';
+
+import MESSAGES from '../messages';
+
+type OperationType =
+    | 'WRITE_PROFILE'
+    | 'RESET'
+    | 'READ'
+    | 'WRITE_RECORD'
+    | 'CHANGE_STATUS';
+
+export const useGetOperationsTypes = (): Array<DropdownOptions<string>> => {
+    const { formatMessage }: { formatMessage: IntlFormatMessage } =
+        useSafeIntl();
+    return [
+        {
+            value: 'WRITE_PROFILE',
+            label: formatMessage(MESSAGES.writeProfile),
+        },
+        {
+            value: 'RESET',
+            label: formatMessage(MESSAGES.reset),
+        },
+        {
+            value: 'READ',
+            label: formatMessage(MESSAGES.read),
+        },
+        {
+            value: 'WRITE_RECORD',
+            label: formatMessage(MESSAGES.writeRecord),
+        },
+        {
+            value: 'CHANGE_STATUS',
+            label: formatMessage(MESSAGES.changeStatus),
+        },
+    ];
+};
```

Comment:
Do you have to wrap it in a memo or is `useSafeIntl` enough to guarantee that the translations will be updated? I'm not honestly not sure anymore. (Since you can't switch languages from the page your calliing this hhok from it doesn't change much, I just wonder)